### PR TITLE
Add some return type hints for methods of Werkzeug's EnvironBuilder c…

### PR DESCRIPTION
**Pull request by adamtheturtle**
…lass.

These are directly copied from the hints in the source code at
https://github.com/pallets/werkzeug/blob/e23971df0d1fea0876a75dd804551c858df7e2fd/src/werkzeug/test.py.

### CODE REVIEW

File: third_party/2and3/werkzeug/test.pyi [hunk 1]
```diff
@@ -74,12 +74,12 @@ class EnvironBuilder:
...
+    def server_name(self) -> str: ...
     @property
-    def server_port(self): ...
-    def __del__(self): ...
-    def close(self): ...
-    def get_environ(self): ...
+    def server_port(self) -> int: ...
+    def __del__(self) -> None: ...
+    def close(self) -> None: ...
+    def get_environ(self) -> Dict[str, Any]: ...
```
Comment (srittau) [inline third_party/2and3/werkzeug/test.pyi:82]: We should use `WSGIEnvironment` here.

**Review by srittau — CHANGES_REQUESTED** [review]
Thanks, one minor suggestion below.

**Comment by adamtheturtle** [comment]
@srittau Thank you for the quick review. I have addressed the comment.

**Review by srittau — APPROVED** [review]

**Comment by srittau** [comment]
FWIW, I've opened https://bugs.python.org/issue42012, where I suggest to add `WSGIEnvironment` et al. to the stdlib, so that werkzeug & co. can use it directly in the future.

**Pull request merged**